content/sensu-go/5.8/reference/backend.md

```diff
@@ -653,6 +654,22 @@ sensu-backend start --no-embed-etcd
 # /etc/sensu/backend.yml example
 no-embed-etcd: true{{< /highlight >}}
 
+
+| etcd-cipher-suites    |      |
```

Comment:
Is there a default value?

Reply:
No - an empty value will lead to the default etcd cipher suites being used. (I'm not sure what they are.)

Reply:
Based on [this doc](https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/security.md): "empty will be auto-populated by Go". So maybe the values listed [here](https://golang.org/pkg/crypto/tls/#pkg-constants) would be the defaults?

Reply:
That appears to be the case.

https://github.com/etcd-io/etcd/blob/master/embed/config.go#L187